src/pharmpy/tools/common.py

```diff
@@ -65,9 +65,10 @@ def create_results(
     rank_type,
     cutoff,
     bic_type='mixed',
+    strictness="minimization_successful or (rounding_errors and sigdigs >= 0)",  # FIXME: set default to None
```

Comment:
Why is this FIXME here?

Reply:
Because the default strictness should be in the tool arguments. But strictness is not implemented in all tools so we need the default here as well. But once strictness is implemented in all tools we can remove the default here. Otherwise we have the default twice.